Sources/ScipioKit/Producer/Cache/CacheSystem.swift

```diff
@@ -86,16 +86,16 @@ extension PinsStore.PinState: Hashable {
 }
 
 public struct CacheKey: Hashable, Codable, Equatable {
-    var targetName: String
-    var pin: PinsStore.PinState
+    public var targetName: String
+    public var pin: PinsStore.PinState
```

Comment:
Why did you change to public?

Reply:
Because I want to call from external storages.